docs/developer_guide/getting_started.md

```diff
@@ -9,27 +11,25 @@ In addition to traditional RPC wallet communication, monero-javascript can manag
 ![Monero-javascript hierarchy](../architecture.png?raw=true)*Monero-javascript manipulates XMR and communicates with the blockchain via three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
```

Comment:
lowercase

Reply:
"manipulates" word choice.  Does this have updates from @trasherdk reviews?

Reply:
I agree. `manipulate` is a bad choice. I just can't think of a good expression just now.

Reply:
Something screwy happened in my struggles wrestling with git. Git continues to defy and evade my attempts to understand and control it. I had already fixed this once. A lot of my work got reverted to a previous state somehow.